### PR TITLE
fix: update hashes of terraform actions

### PR DESCRIPTION
**Pull request by konturn**
cc @areusch @Mousius apparently, the hashes of the actions which are published for each tag differ from the tag's corresponding SHA. I inferred these SHA's by running pipelines on my fork (pinning the actions by tag) and observing the SHA's that are actually used. Surely there's a better way to do this, but it shouldn't matter until we decide to update the Actions sometime in the (probably distant) future.

### CODE REVIEW

File: .github/workflows/apply.yml
Comment (areusch) [inline .github/workflows/apply.yml]: can you include the tag name in a comment?

**Review by areusch — COMMENTED** [review]
oh ok i'm fine with this too if it wasn't too much effort. one request

File: .github/workflows/apply.yml
Comment (konturn) [reply .github/workflows/apply.yml]: done.

**Comment by konturn** [comment]
Another issue not addressed here is that checks are failing since the PR can't access the actions secrets (which contain the AWS static credentials) needed to run a plan. Not an issue for people with write access to the repository, since they will be able to open MR's against the repo directly without going through a fork, and the plan should work in that context. Currently mulling over methods to allow external contributors to use the CI as well.

**Review by areusch — APPROVED** [review]
thanks @konturn, @Mousius any comments?

**Review by Mousius — APPROVED** [review]
Can't hold this back anymore!

**Pull request merged**
